packages/components/src/products.tsx

```diff
@@ -244,12 +243,10 @@ export const PRODUCTS_MENU_LIST: MenuItem['items'] = Object.fromEntries(
           type: 'separator',
           title: (
             <>
-              {/* This is a one-off class, because I want to style the parent. */}
+              {/* This is a one-off class. The margins and paddings of the parent list item are were large. */}
               {/* eslint-disable-next-line tailwindcss/no-custom-classname */}
-              <style className="label-separator">
-                {
-                  'li:has(.label-separator) { margin: 0.75rem 0 0.25rem 0 !important; padding: 0 !important; }'
-                }
+              <style className="hive-label-separator">
+                {'li:has(>.hive-label-separator) { margin: 0.75rem 0 0.25rem 0; padding: 0 }'}
               </style>
```

Comment:
is there a way to avoid adding `<style>`? I am not fun of it

Reply:
So I want to override the style of the parent. Is it possible in `nextra-theme-docs`? Like passing a `className`? This would be a pretty okay Tailwind class.

Reply:
ok let's keep it as is for now